src/colorizer/Dataset.ts

```diff
@@ -17,7 +17,7 @@ type DatasetManifest = {
   times?: string;
 };
 
-type FeatureData = {
+export type FeatureData = {
```

Comment:
This export is no longer used... though it's super possible that it will be later, so it might as well stay